include/pistache/pist_timelog.h

```diff
@@ -227,7 +234,15 @@ class __PS_TIMEDBG
             va_end(ap);
 
             if (ln >= ((int) sizeof_buf))
+            {
+                #ifdef _IS_BSD
+                strlcat(buf_ptr, "...", 5);
+                #elif defined(__linux__)
+                strncat(buf_ptr, "...", 5);
+                #else
                 strcat(buf_ptr, "...");
+                #endif
```

Comment:
Same as above

Reply:
Addressing in the same fashion as strlcpy